lessons/lesson1.md

```diff
@@ -1,5 +1,28 @@
-## Lesson 1: Bootstrap
+## Lesson 1: Create a Console Integration 
 
-Bootstrap your app.
+In order to use custom event SDK, you need to get below informtion from console integration.
 
-Next lesson: [Creating your app](lesson2.md)
\ No newline at end of file
+- `IMS Org Id`: The Organization Id for which the provider, event metadata, etc are to be created which can be obtained using the Console or Transporter API.
+- `API key`: The API Key ( client id ) for the integration ( project workspace ) 
+- `JWT Token`: Note that I/O Management Service needs to be enabled for the integration
+- `Config.zip`: Config file downloaded from console including private key and certificate_pub.crt
+- `project.json`: for instance, `projectname-orgId-Production.json` file downloaded from console 
+
+
+1. Navigate to Adobe I/O console at [https://console.adobe.io](https://console.adobe.io) in your browser and create a project or using your exsiting project 
+2. Select `Add to Project` -> `Add an API` -> `Adobe Services` -> `I/O managemenet API`
+![add-api](assets/add-api.png)
+
+3. follow the steps to configure API, create a new service account (JWT) credential, `config.zip` will be
+downloaed automatically, you will need the private key to generate JWT token 
+
+4. Go to `project overview` tab, download project metadata from below `download` button and get the needed info from this `.json`file 
+![download](assets/download.png)
+![projectjson](assets/project-json.png)
```

Comment:
The annotations of Project ID etc. is too small, font should be bigger.
See rendered content at https://init-content--adobeio-codelabs-custom-event--adobedocs.project-helix.page/?src=/lessons/lesson1.html